huluti/altcha-bundle/1.0/config/routes/huluti_altcha.yaml

```
@@ -0,0 +1,2 @@
+live_component:
```

Please update routes import, as it should be named after your bundle. Additionally, `live_component` is reserved for ux live component package.

```suggestion
huluti_altcha:
```

---

done. thanks